nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -603,50 +578,33 @@ def forward(
 
         return transf_log_probs, encoded_len, enc_states, enc_mask
 
-    def compute_loss(
-        self, batch: tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor] | None
-    ) -> torch.Tensor:
-        """
-        Run forward pass through the model and compute the loss.
-
-        Args:
-            batch: a tuple of 4 tensors (signal, signal_len, tokens, tokens_len) as returned
-                by :class:`~nemo.collections.asr.data.audio_to_text_lhotse_prompted.PromptedAudioToTextLhotseDataset`.
-                When batch is ``None``, we'll return a zero tensor.
-        Returns:
-            The computed loss value as a single-element tensor.
-        """
+    # PTL-specific methods
+    def training_step(self, batch, batch_nb):
+        signal, signal_len, transcript, transcript_len = batch
 
         if batch is None:
-            return torch.tensor([0.0])
+            tensorboard_logs = {
+                'train_loss': torch.Tensor[0.0],
+                'learning_rate': self._optimizer.param_groups[0]['lr'],
+            }
+            return {'loss': audio_loss, 'log': tensorboard_logs}
 
-        signal, signal_len, transcript, transcript_len = batch
         input_ids, labels = transcript[:, :-1], transcript[:, 1:]
-
-        transf_log_probs, encoded_len, enc_states, enc_mask = self.forward(
+        transf_log_probs, _, _, _ = self.forward(
```

Comment:
may be keep encoded_len, enc_states, enc_mask, instead of _, _, _ so that it is clear what is being returned? (even if we don't use them) ?